src/main/java/core/basesyntax/service/impl/ReportWriterImpl.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Map;
+
+public class ReportWriterImpl implements ReportWriter {
+    private static final String CSV_SPLITTER = ",";
+    private static final String FRUIT_HEADER = "fruit";
+    private static final String QUANTITY_HEADER = "quantity";
+
+    @Override
+    public void writeReport(String path) {
+        writeToFile(Path.of(path), createReport());
+    }
+
+    private String createReport() {
+        StringBuilder reportBuilder = new StringBuilder();
+        reportBuilder.append(FRUIT_HEADER)
+                .append(CSV_SPLITTER)
+                .append(QUANTITY_HEADER)
+                .append(System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : Storage.fruits.entrySet()) {
```

Comment:
Don`t use Storage directly. It`s bad practice. Create StorageDao class, and work with Storage through him.